pandas-stubs/core/series.pyi

```diff
@@ -1886,6 +1886,7 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     def set_axis(
         self, labels, *, axis: Axis = ..., copy: _bool = ...
     ) -> Series[S1]: ...
+    def iteritems(self) -> Iterable[tuple[Hashable, S1]]: ...
```

Comment:
Can you please also add two tests for these two methods?

Reply:
@twoertwein  The tests have been added.

Reply:
Just noticed that the tests are failing. Will fix soon

Reply:
You can also run the test locally: `poe test`

https://github.com/pandas-dev/pandas-stubs/blob/main/docs/setup.md